tests/main/snapd-update-services/task.yaml

```diff
@@ -0,0 +1,76 @@
+summary: systemd units get regnerated when snapd is refreshed
+
+systems:
+  # there seems to be some kernel issue with duplicated mounts in
+  # /proc/*/mounts on 14.04 so it makes it more difficult to test
+  # there.
+  - ubuntu-16.04-*
+  - ubuntu-18.04-*
+  - ubuntu-2*
+
+prepare: |
+  # Install a snap with a service running
+  service_snap="$("${TESTSTOOLS}/snaps-state" pack-local test-snapd-service)"
+  snap install --dangerous "${service_snap}"
+  systemctl is-active snap.test-snapd-service.test-snapd-service.service
+
+  #. "$TESTSLIB"/prepare.sh
+  unpackdir=/tmp/snapd-update-services
+  snap download --edge snapd --basename=snapd_edge
+  unsquashfs -no-progress -d "${unpackdir}" snapd_edge.snap
+  #rm -f "$UNPACK_DIR"/etc/apparmor.d/*
+  dpkg-deb -x "${GOHOME}"/snapd_*.deb "${unpackdir}"
+  cp /usr/lib/snapd/info "${unpackdir}"/usr/lib/snapd
```

Comment:
Please add a comment explaining what we are doing

Reply:
I copied some code from `prepare.sh` and forgot to clean it up. That line is not useful.

Reply:
Removed.